src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,8 +3,12 @@
 import java.util.Random;
 
 public class Lottery {
-    public static Ball getRandomBall() {
-        return new Ball(ColorSupplier.getRandomColor(),
-                new Random().nextInt(100));
+    private static final int MAX_RANDOM_BALLS = 100;
+
+    public Ball getRandomBall() {
```

Comment:
Each time this method is called, new instances of ColorSupplier and Random are created. Let's think about how we can optimize this logic and create each instance only once.